package.json

```diff
@@ -52,7 +52,6 @@
     "eslint": "5.16.0",
     "eslint-config-canopy": "2.3.0",
     "husky": "^2.2.0",
-    "in-publish": "^2.0.0",
```

Comment:
shouldn't removing this from package.json also impact yarn.lock?

Reply:
Oh yea, have to update that.

Reply:
OK, finally got around to updating the lock file.